src/main/connector/system/Windows/Windows.yaml

```diff
@@ -76,7 +76,7 @@ monitors:
         memoryInformation:
           type: wmi
           namespace: root\CIMv2
-          query: SELECT FreeAndZeroPageListBytes,FreeAndZeroPageListBytes,CacheBytes,CacheBytes FROM Win32_PerfRawData_PerfOS_Memory
+          query: SELECT AvailableBytes,AvailableBytes,CacheBytes,CacheBytes FROM Win32_PerfRawData_PerfOS_Memory

```

Comment:
`AvailableBytes` twice?

Reply:
One becomes usage, the other utilization.